public/app/slider-theme.css

```
@@ -1,430 +1,508 @@
+  animation:             toastIn 0.18s ease-out;
+  pointer-events:        auto;
 }
-
-/* Value readout chip — vibrant */
-.slider-val, .sr-val {
-  font-variant-numeric: tabular-nums;
-  transition: color .15s ease, background .15s ease, border-color .15s ease, box-shadow .2s ease, transform .12s ease;
+.toast-success { border-color: #2ecc71; color: #2ecc71; }
+.toast-error   { border-color: var(--vip-accent); color: var(--vip-accent); }
+.toast-warn    { border-color: var(--vip-rt); color: var(--vip-rt); }
+@keyframes toastIn {
```

Stylelint (17.11.0)

[error] Expected keyframe name "toastIn" to be kebab-case (keyframes-name-pattern)

(keyframes-name-pattern)